test/ConnectionManager.spec.ts

```diff
@@ -349,9 +350,9 @@ describe('ConnectionManager', () => {
         ProviderType.WALLET_CONNECT,
         chainId
       )
-      ;(connector as WalletConnectConnector).walletConnectProvider = getSendableProvider(
-        chainId
-      )
+        ; (connector as WalletConnectConnector).walletConnectProvider = getSendableProvider(
+          chainId
+        )
```

Comment:
I'm guessing the changes in this file are the linter, but this one in particular stands out. Is this actually what prettier wrote?

```ts
    it('should return an instance of WalletConnectConnector supporting all chain ids', () => {
      const connector = connectionManager.buildConnector(
        ProviderType.WALLET_CONNECT,
        chainId
      )
        ; (connector as WalletConnectConnector).walletConnectProvider = getSendableProvider(
          chainId
        )
```

seems...wrong?

Reply:
Oh prettier does that a lot. My brain already ignores it

Reply:
it is actually a known issue of prettier, basically it resolves de `) (` putting a semi-colon in the middle.